.pnpmfile.cjs

```diff
@@ -0,0 +1,115 @@
+// Note if you change something here, you'll have to make a package.json mismatch pnpm-lock.yaml to
+// get it re-run. An easy way to do that is to just edit pnpm-lock.yaml to change the version number
+// of husky near the top.
+
+/**
+ * Fix package dependencies.
+ *
+ * We could generally do the same with pnpm.overrides in packages.json, but this allows for comments.
```

Comment:
I appreciate the ability to add comments. A lot better experience to be able to understand what's there and perhaps when they can be removed. Thanks.